README.md

```diff
@@ -88,6 +91,12 @@ future {
 While this is really short and simple, it is equally limited. In many cases, you will need a way to indicate that the task failed. To do this, instead of returning the value, you can return a Result. Results can indicate either a success or a failure:
 
 ```swift
+enum ReadmeError: ErrorType {
+    case
+    RequestFailed,
+    TimeServiceError
```

Comment:
Could you update the enum definition to the common syntax. The [Swift book](https://developer.apple.com/library/ios/documentation/Swift/Conceptual/Swift_Programming_Language/Enumerations.html#//apple_ref/doc/uid/TP40014097-CH12-ID146) uses either a single line with all cases prefixed by `case` or one case per line with `case` on every line.